Wason’s Selection Task.md

```diff
@@ -0,0 +1,112 @@
+
+**Phase 3.** The subject was invited to project falsity on to the residual cards i.e. those cards which had not been selected. Each pair of cards (representing the same value) was pointed to in turn, starting with the Q' cards, if these had not been selected: “could anything on the back of those cards make the rule false?”
+
+
+## Roles 
+
+Each participant completed the same task following instructions.
+
+## Instructions
+
+Instruction written by CSSLab:
+
+*Experiment 1 - The projection of falsity*
+
+Thank you for participating in this experiment today. We are looking to understand how people making the contrapositive inference from conditional sentences of the form, “if P then Q.” You will be allocated alternately to a group or to work individually. The sentence placed on the desk in front of you is the conditional sentence you will analyze in this experiment. Cards on the desk with letters on their front have numbers on their back and vice versa. Your task is to determine for each of the card whether knowing what is on the other side would enable you to find out whether the conditional sentence is true or false.  
+
+(After the selection is carried out)
+
+To the experimental group:
+"What letter/number on the back of the card do you think would render the sentence false? Or none if you believe none would do so."
+"You are allowed to revise your selection now if you would like to."
+"You are now allowed to turn over each card to evaluate the sentence to true or false."
+
+To the control group:
+"Now you are given some time to think again about your selection and make any adjustment before submitting your final choices."
+
+*Experiment 2 - The restricted contingency programme*
+
+Thank you for participating in this experiment today. We are looking to understand how people making the contrapositive inference from conditional sentences of the form, “if P then Q.” You will be allocated alternately to a group or to work individually. You will be presented with a conditional sentence and four cards that are with a geometric shape on one side and a coloured scribble on the other side. Your task is to select the one card which makes the rule false and any cards that make the rule true. You are allowed to examine the cards briefly before presented with the sentence you need to analyze. 
+
+To the experimental group:
+(During the selection) "Please pick out any cards that would prove the rule true. Please note that your decisions means that the rule only held one way.”
+
+*Also, refer to the three phases mentioned in Steps*
+
+# Criteria
+## Performance calculation
+
+In Experiment 1, participants were considered to be able to make the contrapositive inference from the conditional sentences if they choose Q' to determine whether a given conditional sentence was true of false if they were to know the values on the back of the cards. 
```

Comment:
Maybe change "were" to "are" here so that it is parallel with the next paragraph. Otherwise, everything seems good